authorization/src/main/kotlin/org/modelix/authorization/KtorAuthUtils.kt

```diff
@@ -182,28 +156,14 @@ fun ApplicationCall.getUserName(): String? {
     return principal<AccessTokenPrincipal>()?.getUserName()
 }
 
+@Deprecated("Use ModelixAuthorizationConfig.verifyTokenSignature")
 fun verifyTokenSignature(token: DecodedJWT) {
```

Comment:
Is this still used somewhere after the changes introduced in this PR?

Reply:
No. Removed it.